scripts/build/companion_tools/050-make.sh

```diff
@@ -23,7 +23,7 @@ do_companion_tools_make_build() {
     CT_DoExecLog ALL ${make}
     CT_DoExecLog ALL ${make} install
     if [ "${CT_COMP_TOOLS_make_gmake}" = "y" ]; then
-        CT_DoExecLog ALL ln -sv ${make} "${CT_BUILDTOOLS_PREFIX_DIR}/bin/gmake"
+        CT_DoExecLog ALL ln -sv make "${CT_BUILDTOOLS_PREFIX_DIR}/bin/gmake"
```

Comment:
It will not invoke /usr/bin/make since the directory ${CT_BUILDTOOLS_PREFIX_DIR}/bin is inside $PATH inside the ct-ng build. So it will invoke the make binary inside ${CT_BUILDTOOLS_PREFIX_DIR}/bin which is the correct one.

I tested the change and it worked for me.